fine_tuning_sql/notebooks/1_download_data.ipynb

```diff
@@ -150,7 +159,7 @@
    "source": [
     "## Summary\n",
     "\n",
-    "After this stage we should have 2 jsonl files, one for pretraining and one for fine-tuning."
+    "After this stage we should have 1 to 3 jsonl files in the [data folder](../data/), either for pretraining or for fine-tuning."
```

Comment:
For the final recipe only 2 jsonl files should be used.
1. for training, the full squad
2. fir fine-tuning

Reply:
fixed